service/deploy.py

```diff
@@ -158,10 +158,22 @@ def configure_ansible(logger):
 
 
 def build_host_name(ip):
+    """
+    Return the host name
+    * iPlant or raw
```

Comment:
this is only returning "iplant" - in what case would `raw_hostname` get returned?